doc/nrf/libraries/dfu/dfu_multi_image.rst

```diff
@@ -0,0 +1,41 @@
+.. _lib_dfu_multi_image:
+
+DFU multi-image
+###############
+
+.. contents::
+   :local:
+   :depth: 2
+
+The DFU multi-image library provides an API for writing a DFU multi-image package that is being downloaded in chunks of an arbitrary size.
+The DFU multi-image package is a simple archive file that consists of a CBOR-based header that describes contents of the package, followed by a number of updates images, such as firmware images for different MCU cores.
+Images included in a DFU multi-image package are identified by numeric identifiers assigned by the user.
+The library provides a way for the user to register custom functions for writing a single image with a given identifier.
+Given that the library makes no assumptions with respect to formats of images included in a written package, it makes it a general-purpose solution for device firmware upgrades.
+For example, it can be used to upgrade the :ref:`nRF5340 DK <ug_nrf5340_multi_image>` firmware.
+
+Configuration
+*************
+
+To enable the DFU multi-image library, set the :kconfig:option:`CONFIG_DFU_MULTI_IMAGE` Kconfig option.
+
+To configure the maximum number of images that the DFU multi-image library is able to process, use the :kconfig:option:`CONFIG_DFU_MULTI_IMAGE_MAX_IMAGE_COUNT` Kconfig option.
+
+To enable building the DFU multi-image package that contains common update images, such as the application core firmware, the network core firmware, or MCUboot images, set the :kconfig:option:`CONFIG_DFU_MULTI_IMAGE_PACKAGE_BUILD` Kconfig option.
```

Comment:
```suggestion
To enable building a DFU multi-image package that contains common update images, such as the application core firmware, the network core firmware, or MCUboot images, set the :kconfig:option:`CONFIG_DFU_MULTI_IMAGE_PACKAGE_BUILD` Kconfig option.
```
I'm not sure what the word "common" is doing here? Does it means "shared"?

Reply:
It means "commonly used".  Typically people want to include the application core/network core firmware when they perform the DFU. Should I use "commonly used" or you have a better suggestion?

Reply:
Yes, "commonly used" is much clearer.